fw/http_stream_sched.c

```diff
@@ -313,11 +319,15 @@ void
 tfw_h2_add_stream_dep(TfwStreamSched *sched, TfwStream *stream,
 		      TfwStreamSchedEntry *dep, bool excl)
 {
+	TfwH2Ctx __maybe_unused *ctx = container_of(sched, TfwH2Ctx, sched);
 	u64 deficit, min_deficit;
 	bool stream_has_children;
 
 	tfw_h2_stream_sched_spin_lock_assert(sched);
 
+	T_DBG3("%s: ctx [%px] (stream id %u excl %d)\n",
+	       __func__, ctx, stream->id, excl);
```

Comment:
For `h2load -n 20 -c 1 -t 1 -m 10 https://tempesta-tech.com` I have printed only these lines for stream creation:
```
[   90.905579] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 1 excl 0)
[   90.906441] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 3 excl 0)
[   90.907272] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 5 excl 0)
[   90.908124] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 7 excl 0)
[   90.909186] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 9 excl 0)
[   90.910013] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 11 excl 0)
[   90.911411] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 13 excl 0)
[   90.912171] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 15 excl 0)
[   90.912944] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 17 excl 0)
[   90.913776] [tempesta fw]       tfw_h2_add_stream_dep: ctx [ffff88814fe04ca8] (stream id 19 excl 0)
```
Is it possible to also print which stream the current stream depends on and its weight, like you do this in `tfw_h2_change_stream_dep()`?

In https://tempesta-tech.com/knowledge-base/HTTP2-streams-prioritization/ you had a wonderful messaging: there were not function names, only human friendly text, and useful information about requested resources, sent bytes, streams dependency and weights.

Maybe current deficit would be alos useful